scripts/exrap_dump.sh

```diff
@@ -496,7 +497,7 @@ else
 
 fi
 
-$ushscript_dump/bufr_dump_obs.sh $dumptime ${def_time_window_2} 1 vadwnd \
+$USHbufr_dump/bufr_dump_obs.sh $dumptime ${def_time_window_2} 1 vadwnd \
```

Comment:
This change is not required. The original $uscript_dump is defined in JOBSPROC_RAP_DUMP and can be retained. Please revert back to original code:
$ushscript_dump/bufr_dump_obs.sh $dumptime ${def_time_window_1} 1 vadwnd \

Reply:
revised:
$ushscript_dump/bufr_dump_obs.sh $dumptime ${def_time_window_2} 1 vadwnd \